calico-enterprise/operations/clis/calicoq/installing.mdx

```diff
@@ -10,6 +10,13 @@ import MaintenanceClisCalicoqInstalling from '@site/calico-enterprise/_includes/
 
 You can run `calicoq` on any host with network access to the
 {{prodname}} datastore as either a binary or a container.
+
+:::note
+
+   Ensure that you install the binary on a Linux node. The binary does not work on other nodes.
```

Comment:
1. Contents of an admonition should be indented to align with the `:::`.

2. Generally, unrelated fixes are best submitted in separate PRs. Makes for a cleaner history, and it's easier to undo something if we need.

3. The fix for this belongs in every product/version with calicoq. 

4. Since this is pretty low stakes, I wonder if it would be simpler to just modify line 11:
`You can run `calicoq` on any host with network access to the
{{prodname}} datastore as either a Linux binary or a container.`

5. The note needs SME approval.

Reply:
1) Sure
2) Very true. Will create a separate PR.
3) Sure, will update the other versions/products.
4) I had thought about it, but I dropped the idea because people may overlook it. So I wanted to explicitly mention it in a note.
5) Sure